pontoon/base/models/user.py

```diff
@@ -210,10 +210,14 @@ def user_status(self, locale):
 
 @property
 def contributed_translations(self):
-    """Filtered contributions provided by user."""
-    from pontoon.base.models.translation import Translation
+    """Contributions provided by user."""
+    return self.translation_set.all()
 
-    return Translation.objects.filter(user=self)
```

Comment:
Not directly related to the patch, but using similar pattern as the newly added code.